crates/swc_ecma_codegen/src/lib.rs

```diff
@@ -75,6 +75,56 @@ where
     pub wr: W,
 }
 
+fn replace_close_inline_script(raw: &str) -> Cow<str> {
+    let chars = raw.chars().collect::<Vec<_>>();
```

Comment:
This allocation happens unconditionally, so this function affects performance anyway

Reply:
I use `as_bytes` to avoid this problem